app.js

```diff
@@ -19,10 +19,12 @@ const resources = require('./app/resources')
 const requestHandlers = require('./lib/request_handlers')
 const initRedisClient = require('./lib/redis')
 const initMongoDB = require('./lib/db')
+const initCloudinary = require('./lib/cloudinary')
 const exec = require('child_process').exec
 
 const client = initRedisClient()
 initMongoDB()
+initCloudinary()
```

Comment:
Ahh, I noticed that mongodb and redis both get initialized in a separate file so I assumed that was the "right" thing to do. Would it be better to initialize within `app.js` instead of separating the code to another file?

Reply:
This is fine! Following the same patterns mongoDB and Redis seems preferable.

My issue with `initMongoDB()` is that the function only creates side effects, and now `initCloudinary()` unfortunately does the same thing. And not only that, but because of the way the Cloudinary client library is written, everytime we import the cloudinary library, it's "already" initialized "magically." I don't like it, but I don't think we gain anything by trying to change it.